PROVISIONING-METAL.md

```diff
@@ -106,10 +107,38 @@ Please keep in mind that when using static addresses, DNS information must be su
   * `via` (IP address): Gateway IP address.  If no gateway is provided, a scope of `link` is assumed.
   * `route-metric` (integer): Relative route priority.
 
-Example `net.toml` with comments:
+Version `3` adds in support for bonding and vlan tagging. 
+The support is limited to mode `1` (`active-backup`) for [bonding](https://www.kernel.org/doc/Documentation/networking/bonding.txt).
+Future support may include other bonding options - pull requests are welcome! Version `3` also adds the concept of network devices vs network interfaces.
+Physical devices like ethernet adapters are considered "interfaces", and network devices encompass other kinds of devices like bonds and vlans.
```

Comment:
I kind of went the other way since naming comes into play, essentially you have to give us the exact name of a physical(pre-existing) device, and you have to choose a valid name for the others. So it becomes relevant to know which one you are dealing with.